src/content/docs/en/guides/deploy/space.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Deploy your Astro Site to Space
+description: How to build an Astro site in Space.
+type: deploy
+i18nReady: true
+---
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Deta Space](https://deta.space/developers) is a personal computer that lives in the cloud — a ‘personal cloud’. You can build and run apps for yourself in your own "Space". You can publish the apps you've built, and they'll run for people all around the world. 
+
+You can also install apps published by other devs. You'll get a personal copy of the app in your own cloud, running on your infrastructure. All the data that all these apps save to your Space is for you: to explore, to extend, and to export, if you like.
+
+This guide includes step-by-step instructions for building sites in Space. You can build any type of Astro site in Space: either a static site or a server-rendered site.
+
+## Prerequisite
+
+To push an Astro site to Space, make sure you first:
+
+- Create a [Space account](https://deta.space)
+- Install the [Space CLI](https://deta.space/docs/en/basics/cli) and log in
+
+No matter the type of Astro site you are building, create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
```

Comment:
```suggestion
Create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
```
The beauty is, when your instructions work for all options, you don't even have to say so! It's just the instruction!

Reply:
agree 100%